lib/models/postgrest_response.dart

```diff
@@ -0,0 +1,57 @@
+// To parse this JSON data, do
+//
+//     final postgrestResponse = postgrestResponseFromJson(jsonString);
+
+import 'dart:convert';
+
+import 'package:postgrest/models/postgrest_error.dart';
+
+class PostgrestResponse {
+  PostgrestResponse({
+    this.body,
+    this.status,
+    this.statusText,
+    this.error,
+  });
+
+  dynamic body;
+  int status;
+  String statusText;
+  PostgrestError error;
+
+  PostgrestResponse copyWith({
+    dynamic body,
+    int status,
+    int statusCode,
```

Comment:
this variable is unused. We should remove it.

Reply:
yeah sure ^^